build.sbt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2021 HM Revenue & Customs
+ * Copyright 2025 HM Revenue & Customs
```

Comment:
Can be removed, usually the build.sbt files don't contain copyright headers

Reply:
updated